lte/gateway/python/integ_tests/federated_tests/docker/run.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
```

Comment:
not sure how this python script compares against the following lines of bash. 

```bash
#!/usr/bin/env bash

SNOWFLAKE_FILE='$MAGMA_ROOT/.cache/feg/snowflake'
SNOWFLAKE_DIR="$(dirname "$SNOWFLAKE_FILE")"
rm -rf "$SNOWFLAKE_DIR"
mkdir -p "$SNOWFLAKE_DIR"
touch "$SNOWFLAKE_FILE"

if [[ $# -eq 0 ]] ; then
  docker-compose up -d
else
  docker-compose "$@"
fi
```

Reply:
The only logical difference would be that the bash script deletes the `.cache/feg` directory by default and then creates the path and snowflake file, whereas the python script only does anything regarding the snowflake file if it doesn't exist in the correct way yet. But since it is just a cache it probably doesn't matter in the end.
I made a python script to keep it consistent because that is also what is used for the orc8r to start and stop the containers (it just seems convenient to have the same commands to do the same things for different parts of the project).

Reply:
How about a slightly adapted version:
```bash
#!/usr/bin/env bash

SNOWFLAKE_DIR="$MAGMA_ROOT/.cache/feg"
SNOWFLAKE_FILE="$SNOWFLAKE_DIR/snowflake"

if [[ ! -f "$SNOWFLAKE_FILE" ]] ; then
  if [[ -d "$SNOWFLAKE_FILE" ]] ; then
    rm -rf "$SNOWFLAKE_FILE"
  fi
  mkdir -p "$SNOWFLAKE_DIR"
  touch "$SNOWFLAKE_FILE"
fi

if [[ $# -eq 0 ]] ; then
  docker-compose up -d
else
  docker-compose "$@"
fi
```

Reply:
Both Python or Bash would be fine with me. In this case, the Bash version seems a little easier. On the other hand, we have a `run.py` file for other components too.

Reply:
After discussion we decided to stick with the python script, to stay consistent with different parts of the project.